README.md

```diff
@@ -23,14 +22,26 @@
 -   `DEFAULT_SMTP_TLS_REJECT_UNAUTHORIZED`: SMTP config `tls.rejectUnauthorized` for nodemailer, config would open a connection to TLS server with self-signed or invalid TLS certificate
 -   `DEFAULT_SMTP_AUTH_METHOD`: SMTP config `authMethod` for nodemailer, defines preferred authentication method, defaults to ‘PLAIN’
 -   `DEFAULT_SMTP_DEBUG`: SMTP config `debug` for nodemailer, if set to true, then logs SMTP traffic, otherwise logs only transaction events
+-   `UI_URL`: host for ui
 
 ## FILE STORAGE API
 
+-   `STORAGE`: Can be `s3` or `fs` for files and images uploads
```

Comment:
`STORAGE`: (optional) Can be `s3` or `fs` for files and images uploads

Reply:
This is optional if you use an external server for file storage.

Reply:
`STORAGE`: (optional) if use external server for files and images storage. Can be `s3` or `fs`.